.github/workflows/canary.yml

```diff
@@ -37,18 +43,17 @@ jobs:
       - name: Update version
         run: yarn version --new-version "0.0.0-${GITHUB_SHA::8}" --no-git-tag-version
 
-      - name: Export Solidity - ${{ matrix.export_type }}
-        uses: defi-wonderland/solidity-exporter-action@v2.0.0
+      - name: Export Solidity - Export Type ${{ matrix.export_type }}
+        uses: defi-wonderland/solidity-exporter-action@v2.1.0
         with:
-          # Update package_name with your package name
-          package_name: 'my-cool-project'
+          package_name: ${{ env.PROJECT_NAME }}
```

Comment:
I'd name the var `NPM_PACKAGE_NAME` to make its meaning completely obvious.

Reply:
@0xDiscotech the var is defined in the same action. Not on the .env file

Reply:
@gas1cent in this case, it would append -interfaces if you only get the interfaces or keep the project name only for the other case. Do you think it's still better to have the variable be NPM_PACKAGE_NAME? (As it won't be the actual name for the interfaces)

Reply:
> I'd name the var `NPM_PACKAGE_NAME` to make its meaning completely obvious.

As @0xShaito mentioned, `NPM_PACKAGE_NAME` is not accurate in those cases in which `-interfaces` gets appended to the published package name.